tests/browser_context_options_test.go

```diff
@@ -86,7 +86,13 @@ func TestBrowserContextOptionsExtraHTTPHeaders(t *testing.T) {
 	require.NoError(t, err)
 
 	err = tb.awaitWithTimeout(time.Second*5, func() error {
-		resp, err := p.Goto(tb.url("/get"), nil)
+		opts := &common.FrameGotoOptions{
+			Timeout: common.DefaultTimeout,
+		}
```

Comment:
`page.Goto` could implicitly set a default timeout. But I think this makes it explicit and less error-prone. WDYT?

Reply:
Yep, I was wondering the same. I went with the option of being explicit for now, and ensuring it matched the current behaviour. I think in a future refactor we can make it implicit.